src/components/storage/DiskEncryption.jsx

```diff
@@ -194,6 +194,7 @@ export class Page {
     constructor (isBootIso, storageScenarioId) {
         this.component = DiskEncryption;
         this.id = "disk-encryption";
+        // TODO hide for home-reuse
         this.isHidden = ["mount-point-mapping", "use-configured-storage"].includes(storageScenarioId);
```

Comment:
Just add the scenario id here in the list.

Reply:
This is not so easy, the disk encryption screen contains application and reset of AUTOMATIC partitioning (which is also /home reuse) so it needs more work (I have it in the TODO list) and maybe some smaller refactoring.

Reply:
Trying to address in https://github.com/rhinstaller/anaconda-webui/pull/448 with the issues described below https://github.com/rhinstaller/anaconda-webui/pull/424#issuecomment-2367439972